### DISCUSSION TITLE
Problem with scope_type

### DISCUSSION THREAD

**Discussion by alamakot** [netbox-community/netbox#6691] [Q&A]
Hi,
After upgrade to 2.11 I have problem  with my own address management script.

The problem is with scope_type in vlan_group. I'm trying to add new Vlan Group using VLAN Group Bulk Import but I am getting an error:
"Row 1 scope_type: Select a valid choice. That choice is not one of the available choices."

My input data:
name,slug,scope_type
Example,example,dcim.sites

What am I doing wrong?

I've tried using different variables like:
             ('dcim.region', 'Region'),
            ('dcim.sitegroup', 'Site group'),
            ('dcim.site', 'Site'),
            ('dcim.location', 'Location'),
            ('dcim.rack', 'Rack'),
            ('virtualization.clustergroup', 'Cluster group'),
            ('virtualization.cluster', 'Cluster'),

But still not working.

**Comment by candlerb** [comment]
Note that if using scope_type then you'd also need to provide scope_id.  The first is the type of object and the second is the numeric ID of that object type.

It's annoying because a CSV export shows scope_type as "dcim.site", but that doesn't work in CSV import.  The actual allowed values are supposedly [here](https://github.com/netbox-community/netbox/blob/develop/netbox/ipam/constants.py#L63-L65).

However the following CSV import of vlangroups fails, even if you have a site with ID 1:

```
name,slug,scope_type,scope_id
test,test,site,1
```

I suggest you report it as a bug.

I also notice that the vlangroup upload page shows "site" as an available column.  As far as I can see, that doesn't work at all (it ignored the value I put in there).  So that looks like a second bug.

**Reply by alamakot** [reply]
Thanks for your answer.

I know that scope_type is associated with scope_id, but I didn't know what to type in 'scope_type' because neither value worked...

My script was creating vlan group using name, slug and site but it stopped working after the update :(
